### ISSUE TITLE
It shows failed to load notification

### ISSUE THREAD

**Issue by ChiragSinghal2001** [Harshdev625/SDEverse#191]
It shows a failed to load notification every time. Even if user is not logged in, this need to be fixed, as it make the website little messy

<img width="722" height="347" alt="Image" src="https://github.com/user-attachments/assets/26a72f15-1604-4086-90ce-34403b32ab74" />

I would like to work on it.
Please assign it to me
